2018-07-23 20:17:52,463 util.py[line:55] DEBUG No user defined library files
2018-07-23 20:17:52,467 highlevel.py[line:81] DEBUG Automatically found library files: ['C:\\Windows\\system32\\visa32.dll', 'C:\\Windows\\system32\\visa32.dll']
2018-07-23 20:17:52,476 highlevel.py[line:143] DEBUG Library signatures: 90 ok, 0 failed
2018-07-23 20:17:52,476 highlevel.py[line:116] DEBUG Created library wrapper for C:\Windows\system32\visa32.dll
2018-07-23 20:17:52,526 highlevel.py[line:155] DEBUG viOpenDefaultRM('<ViObject object at 0x026188A0>',) -> 0
2018-07-23 20:17:52,526 highlevel.py[line:1502] DEBUG Created ResourceManager with session 29206136
2018-07-23 20:17:52,641 highlevel.py[line:155] DEBUG viFindRsrc(29206136L, u'?*::INSTR', '<ViObject object at 0x026188A0>', 'c_ulong(2L)', <ctypes.c_char_Array_256 object at 0x02618850>) -> 0
2018-07-23 20:17:52,642 highlevel.py[line:155] DEBUG viFindNext(<ViObject object at 0x026188A0>, <ctypes.c_char_Array_256 object at 0x02618850>) -> 0
2018-07-23 20:17:52,642 highlevel.py[line:155] DEBUG viClose(<ViObject object at 0x026188A0>,) -> 0
2018-07-23 20:17:52,644 highlevel.py[line:155] DEBUG viParseRsrcEx(29206136L, 'GPIB0::15::INSTR', 'c_ushort(1)', 'c_ushort(0)', <ctypes.c_char_Array_256 object at 0x02618990>, <ctypes.c_char_Array_256 object at 0x02618800>, <ctypes.c_char_Array_256 object at 0x026187B0>) -> 0
2018-07-23 20:17:52,645 resource.py[line:201] DEBUG GPIB0::15::INSTR - opening ...
2018-07-23 20:17:52,654 highlevel.py[line:155] DEBUG viOpen(29206136L, 'GPIB0::15::INSTR', <AccessModes.no_lock: 0>, 0, '<ViObject object at 0x026187B0>') -> 0
2018-07-23 20:17:52,654 resource.py[line:224] DEBUG GPIB0::15::INSTR - is open with session 40400736
2018-07-23 20:17:52,654 highlevel.py[line:155] DEBUG viWrite(40400736L, 'CALL:SETup:CHANnel:DOWNlink 10563\r\n', 35, 'c_ulong(35L)') -> 0
2018-07-23 20:17:52,655 messagebased.py[line:305] DEBUG GPIB0::15::INSTR - reading 20480 bytes (last status <StatusCode.success_max_count_read: 1073676294>)
2018-07-23 20:17:54,693 highlevel.py[line:155] DEBUG viRead(40400736L, <ctypes.c_char_Array_20480 object at 0x026187B0>, 20480, 'c_ulong(0L)') -> -1073807339
2018-07-23 20:17:54,694 messagebased.py[line:309] DEBUG GPIB0::15::INSTR - exception while reading: VI_ERROR_TMO (-1073807339): Timeout expired before operation completed.
